virt/virtual_machines/virt-controlling-vm-states.adoc

```diff
@@ -8,11 +8,7 @@ toc::[]
 
 You can stop, start, restart, and unpause virtual machines from the web console.
 
-[NOTE]
-====
-To control virtual machines from the command-line interface (CLI), use the
-xref:../../virt/virt-using-the-cli-tools.html#virt-using-the-cli-tools[`virtctl` client].
-====
+You can use xref:../../virt/virt-using-the-cli-tools.html#virt-using-the-cli-tools[`virtctl`] to manage virtual machine states from the CLI. The `virtctl` command provides more options than the web console, for example, force stop, migrate, or exposing a port.
```

Comment:
- I think the second sentence needs to be tweaked a bit for clarity/grammar.
- My specific suggestion is inspired by my fear that a statement like "...provides more options than the web console, for example..." will not always be accurate.

```suggestion
You can use xref:../../virt/virt-using-the-cli-tools.html#virt-using-the-cli-tools[`virtctl`] to manage virtual machine states from the CLI. For example, you can use `virtctl` to force stop a VM or expose a port. For some actions, `virtctl` provides more options than the web console.
```